6-addmaterialcomponents/firebase_counter/lib/app_component.html

```diff
@@ -1,9 +1,9 @@
 <material-fab (trigger)="dislike()">
-    <glyph icon="thumb_down"></glyph>
+    <material-icon icon="thumb_down"></material-icon>
```

Comment:
Does this work without `materialIcon` in the `directives` list?

Reply:
@kwalrath it worked :-D but ok, lets rather put it there (I am going to do that) :-D

Reply:
That's weird that it worked! It wasn't working for me, and I didn't see a change in angular_components. I'm going to look into this a bit.

Reply:
My bad, it looks like MaterialIconComponent made it into [materialDirectives](https://webdev.dartlang.org/api/angular_components/angular_components/materialDirectives-constant) in time for 0.6.0.